File: AngularApp/src/app/models/experienceItem.ts
export class experienceItem {
    Title:string;
    Subtitle:string;
    TimeSpan:string;
    Summary:string;

    constructor(title:string, subtitle:string, timespan:string, summary:string){
        this.Title = title;
        this.Subtitle = subtitle;
        this.TimeSpan = timespan;
        this.Summary = summary;
    }

    static FromJsonString(jsonElement: { Title: string; SubTitle: string; Timespan: string; Summary: string; }) : experienceItem{
        return new experienceItem(jsonElement.Title, jsonElement.SubTitle, jsonElement.Timespan, jsonElement.Summary);
    }
}

File: TGC.Site.API/TGC.Site.API/Models/DTOs/ExperienceItem.cs
using TGC.AzureTableStorage;

namespace TGC.Site.API.Models.DTOs;

[TableItem("Experiences")]
public class ExperienceItem : AzureTableItem
{
	public string? Title { get; set; }
	public string? SubTitle { get; set; }
	public string? Timespan { get; set; }
	public string? Summary { get; set; }
}
